src/utils/APIBreaks/3.0.0.ts

```diff
@@ -94,4 +94,19 @@ export const breakInfos300: APIBreakInfo[] = [
       'It has been automatically replaced with "ns.ui.setTailTitle()".\n\n',
     showPopUp: false,
   },
+  {
+    brokenAPIs: [
+      {
+        name: "ns.corporation.setAutoJobAssignment",
+        migration: {
+          searchValue: "ns.corporation.setAutoJobAssignment",
+          replaceValue: "ns.corporation.setJobAssignment",
```

Comment:
Do you think we can loosen this to just the bare name? That way, it will catch the cases where people assign namespaces to helper variables. This depends on there not being any other uses of this word, but I think that's the case here.
```suggestion
          searchValue: "setAutoJobAssignment",
          replaceValue: "setJobAssignment",
```

Reply:
That's a good idea.